app/src/main/kotlin/org/fossify/clock/activities/MainActivity.kt

```diff
@@ -179,7 +179,7 @@ class MainActivity : SimpleActivity() {
     private fun storeNewAlarmSound(resultData: Intent) {
         val newAlarmSound = storeNewYourAlarmSound(resultData)
 
-        when (binding.viewPager.currentItem) {
+        when (getTabIndex(binding.viewPager.currentItem)) {
```

Comment:
This is incorrect. The `binding.viewPager.currentItem` property already returns a zero-based index.

Custom audio files don't work because of this.